modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ImportTranslationDisplayContext.java

```diff
@@ -48,6 +48,10 @@ public ImportTranslationDisplayContext(
 		_title = title;
 	}
 
+	public Object fileExtensions() {
+		return ".xliff, .xlf, .zip";
+	}
+
```

Comment:
This is a bit weird :-)

Why not create a constant with the supported extensions and then return a String representation of that?
```
public static final String[] IMPORT_TRANSLATION_SUPPORTED_EXTENSIONS = {
    ".xliff", ".xlf", ".zip"
};
```
```
public String getSupportedFileExtensions() {
    return StringUtil.merge(
        FooConstants.IMPORT_TRANSLATION_SUPPORTED_EXTENSIONS, StringPool.COMMA_AND_SPACE);
}
```

Reply:
Alternatively, instead of creating a constant, embed those extensions in the lang key value; we don't really use those for validation so it's only informative.